packages/main/src/ToolbarButton.ts

```diff
@@ -0,0 +1,112 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+
+import ButtonDesign from "./types/ButtonDesign.js";
+
+import ToolbarItem from "./ToolbarItem.js";
+import ToolbarButtonTemplate from "./generated/templates/ToolbarButtonTemplate.lit.js";
+import ToolbarPopoverButtonTemplate from "./generated/templates/ToolbarPopoverButtonTemplate.lit.js";
+
+@customElement({
+	tag: "ui5-toolbar-button",
+})
+
+/**
+ * @event
+ */
+@event("click", {})
+
+/**
+ * @class
+ * The <code>ui5-toolbar-button</code> represents an abstract action,
+ * used in the <code>ui5-toolbar</code>.
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias ToolbarButton
+ * @extends UI5Element
+ * @public
+ */
+class ToolbarButton extends ToolbarItem {
+	/**
+	 * Defines if the action is disabled.
+	 * <br><br>
+	 * <b>Note:</b> a disabled action can't be pressed or focused, and it is not in the tab chain.
+	 *
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean;
+
+	/**
+	 * Defines the action design.
```

Comment:
Use the established [form of documenting enums](https://github.com/SAP/ui5-webcomponents/blob/main/packages/main/src/Avatar.ts#L132)

```html
 Available options are:
	 * <ul>
	 * <li><code>Never</code></li>
	 * <li><code>Always</code></li>
	 * <li><code>Default</code></li>
	 * </ul>
```